app/frontend/wallet/backend-types.ts

```diff
@@ -105,8 +88,15 @@ export type PoolRecommendationResponse = StakePoolInfoExtended & {
   isInRecommendedPoolSet: boolean
 }
 
+export type TokenObject = {
```

Comment:
the `Object` suffix is IMO too cryptic, ideally we should have a single type using solely BigInts, but meanwhile fr the sake of clarity I'd use an explicit name like `TokenWithStringifiedQuantity`

Reply:
I plan to refactor the types before the release, as we talked, add the cborized prefix and remove the _, I ll include this as well :+1: